components/contribution-flow/index.js

```diff
@@ -140,8 +144,9 @@ class ContributionFlow extends React.Component {
       stepDetails: {
         quantity: 1,
         interval: props.fixedInterval || getDefaultInterval(props.tier),
-        amount: props.fixedAmount || getDefaultTierAmount(props.tier),
+        amount: this.props.verb === 'crypto' ? 5 : props.fixedAmount || getDefaultTierAmount(props.tier),
```

Comment:
What's this?

Why `this.props` vs `props`?

Reply:
Ops, my mistake. 😢 Just corrected it.